### ISSUE TITLE
Unhandled characters in build info arg: 'source-charset:utf-8'

### ISSUE THREAD

**Issue by Tamashh** [camden-smallwood/pdb-decompiler#18]
i seem to be getting this error and i don't know how to solve it here is the log:
```
Running `target/release/pdb-decompiler --out=outp --pdb=fnaf9-Win64-Shipping.pdb`
thread 'main' panicked at 'Unhandled characters in build info arg: 'source-charset:utf-8'; Data: "-Zc:inline -nologo -Oi -c -Gw -Gy -Zm1000 -wd4819 -D_CRT_STDIO_LEGACY_WIDE_SPECIFIERS=1 -D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS=1 -D_DISABLE_EXTENDED_ALIGNED_STORAGE -source-charset:utf-8 -execution-charset:utf-8 -Ob2 -Ox -Ot -GF -errorreport:prompt -D_HAS_EXCEPTIONS=0 -Z7 -MD -bigobj -fp:fast -Zo -Zp8 -we4456 -we4458 -we4459 -wd4463 -we4668 -wd4244 -wd4838 -ID:\BuildAgent\work\FNAF9\Engine\Source -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Launch\Private -ID:\BuildAgent\work\FNAF9\Engine\Source\Developer\DerivedDataCache\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\TraceLog\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Core\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\CoreUObject -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\CoreUObject\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\Engine -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Engine\Classes -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Engine\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\NetCore -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Net -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Net\Core\Classes -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Net\Core\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\ApplicationCore\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\RHI\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Json\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\SlateCore -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\SlateCore\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\InputCore -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\InputCore\Classes -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\InputCore\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\Slate -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Slate\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\ImageWrapper -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\ImageWrapper\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Messaging\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\MessagingCommon\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\RenderCore\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Analytics -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Analytics\AnalyticsET\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Analytics\Analytics\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Sockets\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Net\Common\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\AssetRegistry -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\AssetRegistry\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\EngineMessages -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\EngineMessages\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\EngineSettings -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\EngineSettings\Classes -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\EngineSettings\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\SynthBenchmark\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\Renderer -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Renderer\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\GameplayTags -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\GameplayTags\Classes -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\GameplayTags\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\DeveloperSettings -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\DeveloperSettings\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\PacketHandler -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\PacketHandlers -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\PacketHandlers\PacketHandler\Classes -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\PacketHandlers\PacketHandler\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\PacketHandlers\ReliabilityHandlerComponent\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\AudioPlatformConfiguration -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\AudioPlatformConfiguration\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\MeshDescription -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\MeshDescription\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\StaticMeshDescription -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\StaticMeshDescription\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\PakFile\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\RSA\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\NetworkReplayStreaming -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\NetworkReplayStreaming\NetworkReplayStreaming\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\PhysicsCore -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\PhysicsCore\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\Chaos -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Experimental -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Experimental\Chaos\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Experimental\ChaosCore\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\Intel -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Experimental\Voronoi\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\SignalProcessing\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\AudioExtensions -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\AudioExtensions\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\AudioMixerCore\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\PropertyAccess -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\PropertyAccess\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\ClothingSystemRuntimeInterface -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\ClothingSystemRuntimeInterface\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\AudioMixer -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\AudioMixer\Classes -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\AudioMixer\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Developer -ID:\BuildAgent\work\FNAF9\Engine\Source\Developer\TargetPlatform\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\AnimationCore -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\AnimationCore\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\MoviePlayer -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\MoviePlayer\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Networking\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Projects\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\SandboxFile\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\Serialization -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Serialization\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Cbor\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\Overlay -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Overlay\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\PreLoadScreen\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\InstallBundleManager\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\HeadMountedDisplay -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\HeadMountedDisplay\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\AugmentedReality -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\AugmentedReality\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\MRMesh -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\MRMesh\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\MediaUtils -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\MediaUtils\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Media\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\ClothingSystemRuntimeNv -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\ClothingSystemRuntimeNv\Public -ID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Inc\ClothingSystemRuntimeCommon -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\ClothingSystemRuntimeCommon\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Developer\AutomationController\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Developer\TaskGraph\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\SlateNullRenderer\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\SlateRHIRenderer\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\Launch\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\Runtime\SessionServices\Public -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\PxShared\include -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\PxShared\include\cudamanager -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\PxShared\include\filebuf -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\PxShared\include\foundation -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\PxShared\include\pvd -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\PxShared\include\task -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\PhysX_3.4\Include -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\PhysX_3.4\Include\cooking -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\PhysX_3.4\Include\common -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\PhysX_3.4\Include\extensions -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\PhysX_3.4\Include\geometry -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\APEX_1.4\include -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\APEX_1.4\include\clothing -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\APEX_1.4\include\nvparameterized -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\APEX_1.4\include\legacy -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\APEX_1.4\include\PhysX3 -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\APEX_1.4\common\include -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\APEX_1.4\common\include\autogen -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\APEX_1.4\framework\include -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\APEX_1.4\framework\include\autogen -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\APEX_1.4\shared\general\RenderDebug\public -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\APEX_1.4\shared\general\PairFilter\include -ID:\BuildAgent\work\FNAF9\Engine\Source\ThirdParty\PhysX3\APEX_1.4\shared\internal\include -I"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\INCLUDE" -I"C:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.2\include\um" -I"C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\ucrt" -I"C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\shared" -I"C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um" -I"C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\winrt" -FID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Shipping\Engine\SharedPCH.Engine.ShadowErrors.h -YuD:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Shipping\Engine\SharedPCH.Engine.ShadowErrors.h -FpD:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Shipping\Engine\SharedPCH.Engine.ShadowErrors.h.pch -FID:\BuildAgent\work\FNAF9\fnaf9\Intermediate\Build\Win64\fnaf9\Shipping\Launch\Definitions.Launch.h -TP -GR- -W4 -showIncludes -X"', src/cpp/module.rs:1211:32
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by camden-smallwood** [comment]
This has been fixed in commit [9a8ca7e](https://github.com/camden-smallwood/pdb-decompiler/commit/9a8ca7e80a0b06b2447924daded58b03f366e032).

**Comment by camden-smallwood** [comment]
This has been fixed in commit [05a72dc](https://github.com/camden-smallwood/pdb-decompiler/commit/05a72dc754d3e0c6dab37047ba36201be0840117).

**Comment by Tamashh** [comment]
Thanks again but i got another one:
```
Running `target/release/pdb-decompiler --out=outp --pdb=fnaf9-Win64-Shipping.pdb`
thread 'main' panicked at 'Unexpected character in build info arg: 'J...'; Data: "-c -Zi -JMC -nologo -W3 -WX- -diagnostics:classic -O2 -Ob2 -Oi -DWIN32 -DNDEBUG -D_LIB -D_UNICODE -DUNICODE -Gm- -EHs -EHc -MDd -GS -Gy -fp:precise -Zc:wchar_t -Zc:forScope -Zc:inline -Gd -TC -FC -errorreport:prompt -ID:\Dev-Enterprise\Engine\Source\ThirdParty\MikkTSpace\inc -I"C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.16.27023\include" -I"C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.16.27023\atlmfc\include" -I"C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Auxiliary\VS\include" -I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.10240.0\ucrt" -I"C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Auxiliary\VS\UnitTest\include\v150" -I"C:\Program Files (x86)\Windows Kits\8.1\Include\um" -I"C:\Program Files (x86)\Windows Kits\8.1\Include\shared" -I"C:\Program Files (x86)\Windows Kits\8.1\Include\winrt" -I"C:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\Include\um" -X"', src/cpp/module.rs:1479:28
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by camden-smallwood** [comment]
This has been fixed in commit [f77f31e](https://github.com/camden-smallwood/pdb-decompiler/commit/f77f31e944b9e7e132c3204e20d72f1e8a0eb2ef).

**Comment by Tamashh** [comment]
got something new this time:
```
Running `target/release/pdb-decompiler --out=outp --pdb=fnaf9-Win64-Shipping.pdb`
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/cpp/module.rs:504:44
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by camden-smallwood** [comment]
This is a different issue due to the expected layout of the build info. I have added a safety check in commit [21bee5e](https://github.com/camden-smallwood/pdb-decompiler/commit/21bee5e29758c0fd40e0b18075c1ffeec64df3a0) that gives more information. Could you run it again and post the output?

**Comment by Tamashh** [comment]
here you go:
```
Running `target/release/pdb-decompiler --out=outp --pdb=fnaf9-Win64-Shipping.pdb`
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
thread 'main' panicked at 'Unexpected build info arguments: ["D:\Dev-Enterprise\Engine\Source\ThirdParty\zlib\v1.2.8\lib\Win64-llvm\Release", "D:\Dev-Enterprise\Engine\Source\ThirdParty\zlib\v1.2.8\build\src\zlib-1.2.8\deflate.c"]', src/cpp/module.rs:506:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by camden-smallwood** [comment]
Thanks, I have added a hack fix in commit [39d17e5](https://github.com/camden-smallwood/pdb-decompiler/commit/39d17e51f98d267ae0a58d9fffb61c1d6bf3bdb0), which should fix this issue.

**Comment by Tamashh** [comment]
Thanks but i got another Unexpected character:
```
Running `target/release/pdb-decompiler --out=outp --pdb=fnaf9-Win64-Shipping.pdb`
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
WARNING: failed to parse id 0x0: Support for types of kind 0xffff is not implemented
thread 'main' panicked at 'Unhandled characters in build info arg: 'FD...'; Data: "-O2 -DNDEBUG -MD -DCURL_STATICLIB -ID:\Perforce\ian.fox-third-party\3rdParty\libcurl\intermediate\curl-7.55.1\winbuild -ID:\Perforce\ian.fox-third-party\3rdParty\libcurl\intermediate\curl-7.55.1\lib -ID:\Perforce\ian.fox-third-party\3rdParty\libcurl\intermediate\curl-7.55.1\include -nologo -W4 -wd4127 -EHs -EHc -DWIN32 -FD -c -DBUILDING_LIBCURL -ID:\Perforce\ian.fox-third-party\3rdParty\libcurl\intermediate\deps -ID:\Perforce\ian.fox-ue4-dev-online\Engine\Source\ThirdParty\OpenSSL\1.1.1\Include -ID:\Perforce\ian.fox-ue4-dev-online\Engine\Source\ThirdParty\zlib\v1.2.8\include\Win64\VS2015 -DUSE_OPENSSL -ID:\Perforce\ian.fox-third-party\3rdParty\libcurl\intermediate\deps -DHAVE_ZLIB_H -DHAVE_ZLIB -DHAVE_LIBZ -DUSE_WIN32_IDN -DWANT_IDN_PROTOTYPES -DUSE_IPV6 -Zi -I"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE" -I"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\INCLUDE" -I"C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt" -I"C:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" -I"C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared" -I"C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um" -I"C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\winrt" -TC -X"', src/cpp/module.rs:956:32
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by camden-smallwood** [comment]
This has been fixed in commit [a92e6b9](https://github.com/camden-smallwood/pdb-decompiler/commit/a92e6b9f600ff07638da601d7662d570d5d776de).

**Comment by Tamashh** [comment]
thanks seems to have worked now
